etc/clang-format-all.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+. "$(dirname "${BASH_SOURCE[0]}")/../.evergreen/init.sh"
+
+files="$(find "$LIBMONGOCRYPT_DIR/src" -type f -name '*.c' -o -name '*.h' -o -name '*.cpp' -o -name '*.hpp')"
+readarray -t files <<< "$files"
+bash "$LIBMONGOCRYPT_DIR/etc/clang-format.sh" \
```

Comment:
Can `bash` be omitted when invoking the script to encourage using/respecting shebangs within the file?

Reply:
(Resolving; discussion continuing in other thread: https://github.com/mongodb/libmongocrypt/pull/577#discussion_r1109024061)